test_scripts/Resumption/InteriorVehicleData/022_InteriorVD_resumption_4th_ignition_cycle_absence_resumption.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL does not resume interior vehicle data in 4th ignition cycle
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1)
+--
+-- Sequence:
+-- 1. Three IGN_OFF and IGN_ON cycles are performed
+-- 2. App starts registration with actual hashId after IGN_ON in 4th ignition cycle
```

Comment:
how can you make sure that 
App either not registered during 3 ignitions, or starts registration during any one of 3 ignitions but resumption date does not change and hashId does not change?

Reply:
@yang1070 We can be sure it because current script performs the application registration only in 4-rd ignition cycle.